src/tokens.js

```diff
@@ -2719,4 +2719,1281 @@ module.exports = [
+                                        name: 'tp_button_theme_title_color_tertiary',
+                                        value: '#676d73',
+                                    },
+                                },
+                                format: 'color',
+                            },
+                            {
+                                id: 'button-theme-title-color-caution',
+                                platforms: {
+                                    scss: {
+                                        name: '$tp-button__theme__title-color__caution',
+                                        value: '#ff5a5f',
+                                    },
+                                    javascript: {
+                                        name: 'tpButtonThemeTitleColorCaution',
+                                        value: '#ff5a5f',
+                                    },
+                                    ios: {
+                                        name: 'caution',
+                                        value:
+                                            'UIColor = UIColor(red: 1.0, green: 0.3529412, blue: 0.37254903, alpha: 1.0)',
+                                    },
+                                    android: {
+                                        name: 'tp_button_theme_title_color_caution',
+                                        value: '#ff5a5f',
+                                    },
+                                },
+                                format: 'color',
+                            },
+                            {
+                                id: 'button-theme-title-color-solid',
+                                platforms: {
+                                    scss: {
+                                        name: '$tp-button__theme__title-color__solid',
+                                        value: '#2f3033',
+                                    },
+                                    javascript: {
+                                        name: 'tpButtonThemeTitleColorSolid',
+                                        value: '#2f3033',
+                                    },
+                                    ios: {
+                                        name: 'solid',
+                                        value:
+                                            'UIColor = UIColor(red: 0.18431373, green: 0.1882353, blue: 0.2, alpha: 1.0)',
+                                    },
+                                    android: {
+                                        name: 'tp_button_theme_title_color_solid',
+                                        value: '#2f3033',
+                                    },
+                                },
+                                format: 'color',
+                            },
+                            {
+                                id: 'button-theme-title-color-text',
```

Comment:
@danoc @mallikapotter I made the `text` and `link` button themes iOS-only because I wasn't sure if web/Android used the Button component for text-only buttons/links, but lmk if they do and I can add values for the other platforms.